src/sql/parts/dashboard/common/dashboardPage.component.ts

```diff
@@ -255,15 +256,20 @@ export abstract class DashboardPage extends Disposable implements OnDestroy {
 				}
 
 				let key = Object.keys(v.content)[0];
-				if (key === WIDGETS_TABS) {
+				if (key === WIDGETS_TABS || key === GRID_TABS) {
```

Comment:
It'd be nice to have just a layout_tabs concept for both grid and flow layouts. That way they're able to specialize in terms of how they work.

Also, long term it'll be good if it doesn't just have to be a tab (e.g. you should be able to contribute a sub-grid if needed into a grid widget)